### PR TITLE
[master] Bug 540929: jdbc.sql-cast property does not apply

### PR DESCRIPTION
**Pull request by dazey3**
for #283 
Signed-off-by: Will Dazey <dazeydev.3@gmail.com>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=540929

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=540929

File: jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/property/TestSQLCast.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package org.eclipse.persistence.jpa.test.property;
```
Comment (lukasj) [inline jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/property/TestSQLCast.java]: License header...

Comment (dazey3) [reply jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/property/TestSQLCast.java]: Woops, missed that! Thanks for the review!

**Pull request merged**